openlibrary/plugins/openlibrary/js/readmore.js

```diff
@@ -35,3 +19,22 @@ export function initReadMoreButton() {
         }
     });
 }
+
+export function initClampers() {
+    /*
+      Clamper shows used to show more/less by toggling `hidden`
+      style on parent .clamp tag
+     */
+    $('.clamp').on('click', function(){
+        const up = $(this);
+        if (up.hasClass('clamp')) {
+            up.css({display: up.css('display') === '-webkit-box' ? 'unset' : '-webkit-box'});
+
+            if (up.attr('data-before') === '+ ') {
```

Comment:
I might use a ternary `?` to simplify this since both actions are going to use an `up.attr('data-before')` but just a suggestion, not blocking